cmake/config.cmake.in

```diff
@@ -8,10 +8,10 @@ if (@WITH_OPENSSL@)
     find_dependency(OpenSSL)
 endif()
 
-include(${CMAKE_CURRENT_LIST_DIR}/@name@-targets.cmake)
+include(${CMAKE_CURRENT_LIST_DIR}/hazelcast-cpp-client-targets.cmake)
```

Comment:
`@PROJECT_NAME@` can be used in this file.

Reply:
changed.